src/components/OperationalView/Pane.tsx

```diff
@@ -0,0 +1,206 @@
+import {useRef, useState, useLayoutEffect, useEffect} from "react"
+import Camera from "../Camera";
+import TwoDDriveSim from "../Tools/TwoDDriveSim";
+import noCam from './Main_Camera_Test.png';
+import cam2 from './oatmeal.gif';
+import Menu from "./Menu";
+import Kermit from "../../kermit.jpg";
+import Kermit2 from '../../kermit 2.jpg';
+import Kermit3 from '../../kermit3.gif';
+import Widget from "./Widget";
+
+export default function Pane({rover, paneID, panelLayout, reload}){
+    const [openMenu, setOpenMenu] = useState(false)
+    const [activePane, setActivePane] = useState(null)
+    const [paneType, setPaneType] = useState('cam')
+    const targetRef = useRef(null);
+
+    const [dimension, setDimension] = useState({
+        width: 0,
+        height: 0
+    })
+    //size still needs work;
+    const getSize = () => {
+        const newWidth = targetRef.current.clientWidth;
+        const newHeight = targetRef.current.clientHeight;
+        setDimension({...dimension, width: newWidth, height: newHeight});
+    }
+
+    useEffect(()=>{
+        if(!targetRef.current) return;
+        const resizeObserver = new ResizeObserver(()=> {     
+            getSize();
+        });
+        resizeObserver.observe(targetRef.current);
+        return () => resizeObserver.disconnect();
+    }, []);
+
+    //console.log("pane: ", paneID, " Width: ", dimension.width, " Height: ", dimension.height);
+
+    const toggleMenu = () => {
+      setOpenMenu(true)
+    }
+    var fileName = noCam;
```

Comment:
We should use const/let instead of var. If you need to mutate the variable, declare it with let, otherwise declare it const.